avidemux_plugins/ADM_videoFilters6/zoom/qt4/Q_zoom.h

```diff
@@ -0,0 +1,37 @@
+#ifndef Q_zoom_h
+#define Q_zoom_h
+#include "ui_zoom.h"
+#include "ADM_image.h"
+#include "DIA_flyDialogQt4.h"
+#include "DIA_flyZoom.h"
+#include "zoom.h"
+#if 1
+class Ui_zoomWindow : public QDialog
+{
+	Q_OBJECT
+
+protected: 
+	int lock;
+
+public:
```

Comment:
ctor and dtor need to be public, sure. But it would be great if the stuff which doesn't need to be accessible from outside the class were kept protected or private.

Reply:
copy-paste-merge technology©
blame the crop filter :)